src/utils/content_protection_utils/src/content_protection_utils.c

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file content_protection_utils.c
+ * @brief Provides an implementation of the utilities for interacting with the content protection blocks
+ *
+ * @copyright Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+#include "content_protection_utils.h"
+#include "content_protection_field_defs.h"
+#include <azure_c_shared_utility/strings.h>
+#include <parson.h>
+#include <stdbool.h>
+#include <strings.h>
+
+bool ContentProtectionUtils_DecryptDek(const STRING_HANDLE dek, const STRING_HANDLE dekCryptAlg)
+{
+    return false;
```

Comment:
return false?

Reply:
no code?

Reply:
It's a stub function for our walk phase. Jeff Welden needed the function definitions for some of his integraton work. I need to add the UNREFERENCED_PARAMETERS and will mark it as a stub function